command/issue.go

```diff
@@ -126,10 +128,12 @@ func issueList(cmd *cobra.Command, args []string) error {
 		if userSetFlags {
 			msg = "No issues match your search"
 		}
-		printMessage(colorErr, msg)
+		fmt.Fprintf(colorErr, title(msg))
 		return nil
 	}
 
+	fmt.Fprintf(colorableErr(cmd), title(utils.Pluralize(len(issues), "issue")))
```

Comment:
If the user has passed the `--limit` flag, should this count report the total number of issues? If so, then this will need to be updated to use the count as reported from the API rather than just doing `len(issues)`. Example:

```
$ gh issue list -L 10

23 issues in OWNER/REPO

[ony 10 issues displayed]
```

Reply:
hmm wondering how would that look like if we combine multiple flags?

(assuming there are a total of 40 prs and 10 belong to ShahzadUmair)
```
$ gh pr list -L 5 -a ShahzadUmair

10 pull requests in OWNER/REPO

[only 5 prs displayed]
```
?

Reply:
That's a good point. When it comes to the “No issues/pull requests” line, we made sure to also add “match your search” in case the user has specified some filters. https://github.com/cli/cli/commit/00b38bb11c74ed64a3ce92b33078d8f85fd124fd#diff-1b75ef4be7fc16db780946db34df4021R111

Maybe in the same case, the line could read: “N matching issues/pull requests in OWNER/REPO”?

Reply:
True, it makes sense to display the "match your search" line when there are multiple filters. Will make appropriate changes and share screenshots of how different scenarios look like.

Reply:
no filter or `-L` show
`<total pr count> pull request(s) in OWNER/REPO` 
whereas any other filter or combination of filters show 
`<result pr count> pull request(s) match your search in OWNER/REPO`. 

@mislav @ampinsk do you think it looks okay to show total count in the title but result contains only 5 items (case with `-L` flag or when there are 100 total prs but by default we only get 30)? How would it look like if we were to show something like `30/<total pr count>` somewhere to make it more clear?

<img width="1066" alt="Screenshot 2020-02-19 at 1 40 31 AM" src="https://user-images.githubusercontent.com/26878498/74776288-1e5c1500-52b9-11ea-82d2-affd9f4fe358.png">
<img width="1067" alt="Screenshot 2020-02-19 at 1 38 04 AM" src="https://user-images.githubusercontent.com/26878498/74776299-22883280-52b9-11ea-95cc-2b454f451e3e.png">
<img width="1063" alt="Screenshot 2020-02-19 at 1 37 36 AM" src="https://user-images.githubusercontent.com/26878498/74776302-2320c900-52b9-11ea-82b6-65497f5ec765.png">
<img width="1059" alt="Screenshot 2020-02-19 at 1 37 20 AM" src="https://user-images.githubusercontent.com/26878498/74776305-2320c900-52b9-11ea-865f-bb1512d3a9bd.png">

Reply:
The new output looks great! ✨ 

> do you think it looks okay to show total count in the title when result contains only 5 items (case with `-L` flag or when there are 100 total prs but by default we only get 30 (3rd screenshot))?

Yes, for consistency let's always show the total number of matches for the current query, regardless of implicit or explicit value of `-L`

> How would it look like if we were to show something like `30/<total pr count>` somewhere to make it more clear?

Hmm good suggestion! It had just ocurred to me that people might not realize that the implicit value of `-L` is 30, and that displaying something like “32 pull request in REPO” but then rendering only 30 results might not just be confusing, but might even be perceived as a bug.

I'm in favor of the `<limit>/<total count>` style of output. However, if those numbers are the same, we can drop the slash and display just a single number.

Reply:
The title always shows the total count. In case the the results are not equal to the total count we add a new line (Back to square one 😅) `Showing K/N results`. What do you think of this approach?

<img width="822" alt="Screenshot 2020-02-20 at 1 39 42 AM" src="https://user-images.githubusercontent.com/26878498/74874689-ecaf8080-5382-11ea-8b1d-9f50412cbe38.png">
<img width="712" alt="Screenshot 2020-02-20 at 1 29 51 AM" src="https://user-images.githubusercontent.com/26878498/74874682-e9b49000-5382-11ea-8b9f-980c2d9e9eb7.png">
<img width="675" alt="Screenshot 2020-02-20 at 1 36 28 AM" src="https://user-images.githubusercontent.com/26878498/74874687-ec16ea00-5382-11ea-98bf-e3c5a796f774.png">
<img width="846" alt="Screenshot 2020-02-20 at 1 31 02 AM" src="https://user-images.githubusercontent.com/26878498/74874693-ede0ad80-5382-11ea-9551-893838ee762c.png">

Reply:
Thank you for working on this and offering so many alternatives 😄 I would be interested in what @ampinsk has to say! I would prefer if we could keep just one header line (to keep it simple), but I also see the value in keeping two different sentences especially in combination with the `{N items} match your search in {REPO}` sentence.

Reply:
Hmmm yes thank you for working through this with us, I am struggling with this one! I would also like to keep it to one line, but I do like the clarity this provides. 

What if the top sentence combined these into `3 of 45 pull requests in owner/repo` if we're showing less than the total, and just the plain `45 pull requests in owner/repo` if we're showing the full amount?

Reply:
@mislav @ampinsk I am also in favour of keeping a single line but different scenarios just keep popping up :D 
I think it might be a little confusing if we use `n of k` with the `match your search` title.
`15 of 45 pull requests match your search in Owner/Repo` might come off as if there are only 45 pull requests in the repo. What do you think?

Reply:
Hey @ShahzadUmair sorry I might be confused, I thought 45 would be the total amount in the repo? To be clear, in `n of k` what would `n` represent and what would `k` represent?

Thank you so much for working through all these scenarios! We really appreciate all your work on this 😄

Reply:
And apologies for the confusion there; So when we have no flag or only `-L`, `K` is the total number of PRs in the repo and `N` is the limit in `N of K`.
With any other flag `K` is the total number of matches found in the repo whereas `N` still is the limit.

so, e.g., If a Repo has a total of 100 PRs and 40 open prs have ShahzadUmair as assignee,

`gh pr list` and `gh pr list -L 30` would yield:
```
30 of 100 pull requests in Owner/Repo
```

however, if we add any other flag, e.g., `gh pr list -a ShahzadUmair` or `gh pr list -a ShahzadUmair -L 30 -s open `, output would look something like this:
```
30 of 40 pull requests match your search in Owner/Repo
```
</br>
</br>
Thank you 😊

Reply:
@ampinsk could you please look at above and suggest an alternative?

Reply:
@ShahzadUmair Thanks for working through these scenarios. How about this, using the same scenario you outlined:

With no additional flags other than explicit or implicit limit
```
Showing 30 of 100 pull requests in owner/repo
```

With additional flags:
```
Showing 30 of 40 pull requests in owner/repo that match your search
```

I *think* (and could be wrong) the slight verbiage change makes it properly express what's happening in each case. Let us know if that seems reasonable.

Reply:
thank you @billygriffin. This sounds just right. Just one little addition that I made was to pluralise `match` in case the total count of the PR's that match the result is equal to 1.

`Showing 1 of 1 pull request in owner/repo that matches your search`

Updated the [PR description](https://github.com/cli/cli/pull/440#issue-375304148) with latest screenshots of the current state.
Please let me know if anything seems out of place.

Reply:
Looks really good to me @ShahzadUmair, thanks for updating the PR body with new screenshots to illustrate the scenarios!